web/themes/custom/itkdev/itkdev_project_theme/templates/reports-project-track.html.twig

```diff
@@ -110,14 +111,19 @@
         </div>
       {% endif %}
       {% if track.entity.getType.field_report_type.value is same as('webform_submission') %}
-        {% set tools = data.toolHelper.loadTools(track.entity) %}
-        {% for id, tool in tools %}
-          <div>
-            {% set blockers = data.toolHelper.getToolBlockers(tool) %}
-            {{ include('@itkdev_project_theme/components/webformSubmissionReportHeader.html.twig', {blockers: blockers, tool: tool}) }}
-            {{ drupal_entity('webform_submission', tool.tool_id.value) }}
-          </div>
-        {% endfor %}
+        {% if projectCount is same as (1) %}
```

Comment:
You changing `is same as` to `==` elsewhere in this file and you should do the same here:

```suggestion
        {% if projectCount == 1 %}
```